src/Data/Singletons/Prelude/Identity.hs

```diff
@@ -81,9 +83,7 @@ $(singletonsOnly [d|
     showsPrec d (Identity x) = showParen (d > 10) $
       showString "Identity " . showsPrec 11 x
 
-  -- deriving instance Functor Identity
-  instance Functor Identity where
-    fmap f (Identity x) = Identity (f x)
+  deriving instance Functor Identity
```

Comment:
`Foldable`? `Traversable`?

Reply:
There's a good reason for this one—`Data.Foldable` and `Data.Traversable` must import `Identity`, `Const`, `Sum`, `Product`, `First`, `Last`, and several other newtypes to implement some of its functionality. (For similar reasons, `base` also employs a similar module structure.)